docs/getting-started/supported-languages-and-tools.md

```diff
@@ -1,17 +1,12 @@
 ---
-description: List of all the programming languages that Codacy supports and the corresponding tools that Codacy uses to analyze your source code.
+description: List of tools that Codacy uses to analyze the supported programming languages and frameworks. Codacy generally provides static analysis as well as code duplication, code complexity, and test coverage metrics for each programming language. For cloud infrastructure-as-code frameworks, Codacy provides static analysis to enforce security and compliance best practices.
 ---
 
 # Supported languages and tools
 
-For each supported programming language, Codacy generally provides:
+The table below lists all programming and cloud infrastructure-as-code frameworks that Codacy supports and the corresponding tools that Codacy uses to analyze your source code. Besides this, Codacy uses [cloc](https://github.com/kentcdodds/cloc) to calculate the source lines of code for all supported languages, and supports multiple [test coverage report formats](../coverage-reporter/index.md#generating-coverage).
 
--   Static Analysis
--   Code Duplication
--   Code Complexity
--   Test Coverage
-
-The table below lists all programming languages that Codacy supports and the corresponding tools that Codacy uses to analyze your source code. Besides this, Codacy uses [cloc](https://github.com/kentcdodds/cloc) to calculate the source lines of code for all supported languages, and supports multiple [test coverage report formats](../coverage-reporter/index.md#generating-coverage).
+Codacy generally provides **static analysis** as well as **code duplication**, **code complexity**, and **test coverage** metrics for each programming language. For cloud infrastructure-as-code frameworks, Codacy provides **static analysis** to enforce security and compliance best practices.
```

Comment:
Could "enforce" be too strong? We could use instead "ensure" or "promote", for example.